frontend/src/components/AppLayout.tsx

```diff
@@ -1,64 +1,59 @@
 import Footer from "./Footer";
 import GrantsIdentifier from "./GrantsIdentifier";
 import Header from "./Header";
+import { useTranslations } from "next-intl";
 
 type Props = {
   children: React.ReactNode;
-  // TODO: pass locale into Layout when we setup i18n
-  // locale?: string;
+  locale: string;
 };
 
-const Layout = ({ children }: Props) => {
-  // TODO: Remove during move to app router and next-intl upgrade
+export default function Layout({ children, locale }: Props) {
+  const t = useTranslations();
+
   const header_strings = {
```

Comment:
These strings will be removed once we moved the app router and are no longer using both i18n and next-intl in https://github.com/HHS/simpler-grants-gov/issues/1361